src/components/TodosList.js

```diff
@@ -9,11 +9,15 @@ import TodoItem from './TodoItem';
 
 class TodoList extends PureComponent {
```

Comment:
I have noticed the use of PureCompoent for class components. These risks **prop changes will be ignored If it couldn't find a difference in shallow comparison**. I am sure you use it for the right reasons 😊. Performance 🚀🚀